includes/class-wc-taxjar-integration.php

```diff
@@ -349,6 +350,14 @@ public function get_settings( $current_section = '' ) {
 					'desc'    => __( 'If enabled, TaxJar will calculate all sales tax for your store.', 'wc-taxjar' ),
 					'id'      => 'woocommerce_taxjar-integration_settings[enabled]'
 				) );
+				array_push( $settings, array(
+					'title'   => __( 'Tax Calculation on API Orders', 'wc-taxjar' ),
```

Comment:
Shouldn't `title` and `label` just be the same text?

Reply:
Actually it may be the label is unnecessary. It was utilizing in older versions of Woo and I will check on if it is still necessary for the versions we want to support.

Reply:
@dallendalton did you get more info here?

Reply:
The label field was no longer necessary so it has been removed. Thanks!